.bazelrc

```diff
@@ -351,6 +351,12 @@ build:remote-clang-cl --config=remote-windows
 build:remote-clang-cl --config=clang-cl
 build:remote-clang-cl --config=rbe-toolchain-clang-cl
 
+# Enable auto rustfmt and clippy during build time of Rust source code.
+build:rust-format --aspects=@rules_rust//rust:defs.bzl%rust_clippy_aspect
+build:rust-format --output_groups=+clippy_checks
```

Comment:
this is the officially recommended way to enable this https://bazelbuild.github.io/rules_rust/rust_clippy.html#setup